include/ck/tensor_operation/gpu/device/impl/device_softmax_impl.hpp

```diff
@@ -195,15 +261,32 @@ struct DeviceSoftmaxImpl : public DeviceSoftmax<InDataType,
     {
         const Argument* p_arg_ = dynamic_cast<const Argument*>(p_arg);
 
-        if(!Reduction::IsSupportedArgument(p_arg_))
+        if constexpr(InSrcVectorDim == 0)
         {
-            return false;
-        }
+            if constexpr(NumInvariantDim == 0)
+            {
+                return (false);
```

Comment:
Changed by [8ecb8](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/476/commits/e5859a4155a791dee0c17e25c464a18f1d28ecb8).  Thanks. 

It is my long-term style to use `return(xxx)`, no matter the returned is an express or a single constant or variable.  But clang-format-10 will change it to `return (xxx)`

Reply:
Well I was asking actually because recently I noticed that `decltype(x)` and `decltype((x))` gives two different results as explained [here](https://en.cppreference.com/w/cpp/language/decltype). I understand that in some context, especially when using automatic return type deduction, such style may lead to unwanted behavior.